src/inlineeditorui.js

```diff
@@ -150,6 +155,23 @@ export default class InlineEditorUI extends EditorUI {
 			originKeystrokeHandler: editor.keystrokes,
 			toolbar
 		} );
+
+		// Set toolbar's max-width on the initialization and update it on the editable resize,
+		// if 'shouldToolbarGroupWhenFull' in config is set to 'true'.
+		if ( !this._toolbarConfig.shouldNotGroupWhenFull ) {
+			const widthObserver = getResizeObserver( () => {
```

Comment:
[Since `InlineEditorUIView` gets the configuration anyway](https://github.com/ckeditor/ckeditor5-editor-inline/blob/i/5597/src/inlineeditor.js#L78-L80), maybe let's move this whole observer business there? The class instance knows about the `editableElement` so I think everything's there to move it and the view will take care of itself.

Reply:
Ok, I moved it 👌